### PR TITLE
start migration away from jcenter and gpmaven repos

### PR DESCRIPTION
**Pull request by eighthave**
This enables Maven Central and allows jcenter and gpmaven only for libs that are only on those repos. Ultimately, those libs will have to be moved to Maven Central or replaced.

* jcenter is shutting down in February
* Custom repos like the gpmaven one are not allowed by F-Droid.org since they are a conduit for any kind of code to be included in the app. 

@n8fr8 @bitmold

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,9 +29,21 @@ buildscript {
...
     repositories {
+        mavenCentral()
         google()
-        jcenter()
-        maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" }
         maven { url 'https://jitpack.io' }
+        maven {
+            url "https://raw.githubusercontent.com/guardianproject/gpmaven/master"
+            content {
+                includeGroup('info.pluggabletransports.aptds')
```
Comment (n8fr8) [inline build.gradle:38]: we no longer need this since we are using iptproxy from jitpack

Comment (eighthave) [reply build.gradle:38]: There is still this from that repo:

```gradle
implementation 'info.pluggabletransports.aptds:jsocksAndroid:1.0.4'
```

**Comment by eighthave** [comment]
@tladesignz turns out only v1.2.0 of Volley is on Maven Central, would that work for your Moat code?

**Comment by tladesignz** [comment]
> @tladesignz turns out only v1.2.0 of Volley is on Maven Central, would that work for your Moat code?

Shouldn't be a problem.

**Comment by eighthave** [comment]
as in, just make the change now in this merge requests, and breakage should be 
readily apparent during the next test/release cycle?

**Comment by tladesignz** [comment]
As in: The change log doesn't look like any changes are tangent to our usage at all:
https://github.com/google/volley/wiki/Release-Notes

I had it switched locally, already. Should compile without a problem.

**Comment by eighthave** [comment]
Ok, switched to v1.2.0 from Maven Central.  This needs to be merged after #456 and #454

**Pull request merged**